### PR TITLE
chore: resolve feedback comments

### PR DESCRIPTION
**Pull request by chungquantin**

### CODE REVIEW

File: pallets/nfts/src/lib.rs [hunk 1]
```diff
@@ -1403,7 +1406,7 @@ pub mod pallet {
 		///
 		/// Origin must be the `ForceOrigin`.
 		///
-		/// - `owner`: The account granting approval for delegated transfer.
+		/// - `owner`: The owner of the approvals granted by the `origin`.
```
Comment (Daanvdplas) [inline pallets/nfts/src/lib.rs:1409]: This still doesn't make sense. The owner is not granted by the origin. We can change it back as I can not think of a better setence.

File: pallets/nfts/src/lib.rs [hunk 2]
```diff
@@ -1485,7 +1488,7 @@ pub mod pallet {
 		/// Origin must be `ForceOrigin`.
 		///
 		/// Arguments:
-		/// - `owner`: The account cancelling approval for delegated transfer.
```
Comment (Daanvdplas) [inline pallets/nfts/src/lib.rs:1488]: The owner who granted approval for delegated transfer

File: pallets/nfts/src/tests.rs [hunk 3]
```diff
@@ -2329,48 +2350,58 @@ fn force_cancel_collection_approvals_work() {
...
-			None
-		);
+			)));
 
-		assert_noop!(
-			Nfts::transfer(RuntimeOrigin::signed(delegate), collection_id, item_id, account(4)),
-			Error::<Test>::NoPermission
-		);
+			assert_noop!(
+				Nfts::transfer(
```
Comment (Daanvdplas) [inline pallets/nfts/src/tests.rs:2396]: This is ultimately a test for `transfer`, not for `cancel_approval`

File: pallets/nfts/src/tests.rs [hunk 4]
```diff
@@ -2466,138 +2513,45 @@ fn approvals_limit_works() {
...
-		let (collection, locked_collection) = (0, 1);
-		let item = 42;
+		let (collection_id, locked_collection_id) = (0, 1);
 		let collection_owner = account(1);
-		let item_owner = account(2);
 		let delegate = account(3);
+		let item_id = 42;
+		let item_owner = account(2);
+
+		for origin in [root(), none()] {
```
Comment (Daanvdplas) [inline pallets/nfts/src/tests.rs:2522]: Can we add some comments here to explain these different origin tests.

Would also be good to clarify that this test is for `approve_collection_..` and `force_approve_collection_..`. Same for the other two cases (cancel_collection_approval & clear_collection_approvals)

File: pallets/nfts/src/tests.rs [hunk 5]
```diff
@@ -2206,6 +2263,12 @@ fn cancel_approval_works() {
 		);
 
 		assert_ok!(Nfts::cancel_approval(RuntimeOrigin::signed(account(2)), 0, 42, account(3)));
+		assert!(events().contains(&Event::<Test>::ApprovalCancelled {
```
Comment (Daanvdplas) [inline pallets/nfts/src/tests.rs:2266]: Again, this should not be tested here. Is this event emitted in `check_approval`?

Comment (chungquantin) [reply pallets/nfts/src/tests.rs:2266]: This is inside cancel_approval

**Pull request merged**